dex/asset.go

```diff
@@ -59,19 +59,23 @@ func init() {
 	}
 }
 
-// LockTimeTaker returns the taker locktime value that should be used by both
-// client and server for the specified network. Mainnet uses a constant value
-// while test networks support setting a custom value during build.
+// LockTimeTaker returns the taker locktime value (i.e. how long funds are locked
+// for in swap contract before they can be pulled out - which happens on refund
+// paths) that client and server will use for the specified network. Mainnet uses
+// a constant value while test networks support setting a custom value
+// during build.
```

Comment:
how long funds are locked for in swap contract before they can be ~~pulled out - which happens on refund paths~~ **refunded**

here and below

Reply:
updated